lib/openhab/dsl/types/datetime.rb

```diff
@@ -75,13 +75,13 @@ def initialize(datetime)
         # @return [Integer] -1, 0 or 1 depending on the outcome
         #
         def <=>(other)
+          if other.respond_to?(:zoned_date_time)
```

Comment:
If we changed this to `to_zdt`, it will include TimeOfDay by way of its to_zdt that is implemented in #218 - WDYT?

Reply:
This was mostly to enable comparisons with DateTime, DateTimeItem and GroupItem (if it's a DateTime group), but can see if it can work with TimeOfDay this way as well!

Reply:
It depends on whether we want to ignore the date portion, so today at 7:00 is equal to ToD 7:00 and tomorrow at 7:00 will equal to ToD 7:00 too. If that's the case, then we don't want to use TimeOfDay.to_zdt because that will fill it with today's date.

Reply:
I would think TimeOfDay would mostly (only?) be used with the intent that the represent a time in the current day, so don't think it would be a problem for most users. But might want to inform about that in the docs for the TimeOfDay#to_zdt-method.

Reply:
Marking this as resolved since the other PR for implementing this is to be closed.